### DISCUSSION TITLE
Crossplay v2.1 for Terraria 1.4.4.8.1

### DISCUSSION THREAD

**Discussion by Moneylover3246** [Moneylover3246/Crossplay#46] [Announcements]
## Overview
- Introduces support for servers running on Terraria 1.4.4.8.1
## Changes
- Fixed journey mode clients not being able to join journey mode worlds with the config option enabled

<hr /><em>This discussion was created from the release <a href='https://github.com/Moneylover3246/Crossplay/releases/tag/2.1'>Crossplay v2.1 for Terraria 1.4.4.8.1</a>.</em>

**Comment by palv0** [comment]
this is not working i tried to play whit my mobile friend and its says its a different version

**Reply by EveliaCoss** [reply]
I have the same problem.

[Crossplay] Changing version of index 1 from v1.4.4.5 => v1.4.4.7

148.56.144.99:59530 was booted: You are not using the same version as this server.
